roofit/xroofit/src/xRooNode.cxx

```diff
@@ -5775,9 +5775,9 @@ TH1 *xRooNode::BuildHistogram(RooAbsLValue *v, bool empty, bool errors, int binS
          if (x->hasError())
             h->SetBinError(1, x->getError());
          h->SetMaximum(x->hasMax() ? x->getMax()
-                                   : (h->GetBinContent(1) + std::max(abs(h->GetBinContent(1) * 0.1), 50.)));
+                                   : (h->GetBinContent(1) + std::max<double>(abs(h->GetBinContent(1) * 0.1), 50.)));
```

Comment:
```suggestion
                                   : (h->GetBinContent(1) + std::max(std::abs(h->GetBinContent(1) * 0.1), 50.)));
```

Reply:
I've tried the changes and they also work, thanks for the suggestion!

I've updated the PR now.